app/posts/register-trainee-teachers/2022-04-14-changing-route-and-course-after-registration.md

```diff
@@ -0,0 +1,92 @@
+
+Since January 2022, most of our course change requests are because of the trainee changing from one course to another or one route to another. Sometimes for financial reasons, or sometimes they may decide to learn to teach a different subject.
+
+## Why this has been a challenge for us
+
+The route is the core data item on which much of the record relies. Changing it has been a challenge because of the consequences this has on the record. We only collect the data needed by a route, but if the route changes, we may now be missing the data needed for a complete record. Or worse, the data we have may now be invalid — and we need to delete this invalid data and ask the user to update the record.
+
+For example, if a trainee moves:
+
+* from a provider-led course to a school direct course, then the record will now be missing school details, which are needed to correctly pay funding to the lead school
+* from a course without funding to one with funding available, we should check whether the trainee should be receiving any funding
+
+from a course with a bursary to one that doesn’t have funding, then the existing funding source is invalid
+
+## New designs
+
+In the new designs, the training route is now part of the course details. Previously, it was separate — now we’ll consider it an inherent part of the course. We’ll ask about the route as the first question when setting course details.
+
+### Course details summary card
+
+We’ve combined the existing route and course rows into a single course row. This shows the trainee’s route and relevant course information. If it’s a Publish course, we show the Publish course name and course code. If not, we show the course subjects.
+
+Our Publish course picking page shows just those courses for the current route — so when changing course, we’ll first ask the user to select (or reconfirm) the route.
+
+![The course details summary card with a row that is labelled ‘Course’. Inside the row the route ‘Provider-led’ is shown, together with the the Publish course.](1-new-course-details-summary-card.png "Ammended course row")
+
+## Flow
+
+Selecting ‘change’ on the course row takes the user to our existing route selection page. 
+
+After picking the route we do a number of things:
+
+* if a route does not have Publish courses, we delete any existing Publish information we have on the trainees' course details such as course code and course name.
+* f the route has changed, we check for incompatible course data and delete it. For example deleting study mode when moving to assessment only courses, or deleting Early years subjects when moving away from Early years courses.
+* if the route has Publish courses, we then show the Publish course picking flow.
+* if the route does not have Publish courses, we show the manual course details flow.
```

Comment:
I think this would be clearer if you made it explicit that you're comparing new to old. For example, ‘if ~a~ `the new` route does not have Publish courses, we delete any existing Publish information we have on the trainees' course details such as course code and course name.’